src/core/ext/filters/client_channel/lb_policy/xds/eds.cc

```diff
@@ -56,6 +57,14 @@ constexpr char kEds[] = "eds_experimental";
 
 const char* kXdsLocalityNameAttributeKey = "xds_locality_name";
 
+bool XdsCircuitBreakingEnabled() {
```

Comment:
Please add a TODO indicating that this should be removed as soon as we finish integration testing.